pkgs/by-name/kc/kcl-language-server/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "kcl-language-server";
+  version = "0.1.61";
+
+  src = fetchFromGitHub {
+    owner = "KittyCAD";
+    repo = "kcl-lsp";
+    rev = "v${version}";
+    hash = "sha256-VtrR4v0BJWYdoYFDJpWnmVqDhZMlPGm+g9yjxYfcFxQ=";
+  };
+
+  cargoHash = "sha256-87S+8uyACWx6URGJGcFdfPbH6UJnR+7685zFSof27Do=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
Avoid nested with, whenever possible.

---

meta.changelog is missing

> meta.changelog = "https://github.com/KittyCAD/kcl-lsp/releases/tag/v${version}";

---

done

---

Not done!!!